pkg/asset/installconfig/gcp/validation.go

```diff
@@ -257,6 +262,11 @@ func ValidatePrivateDNSZone(client API, ic *types.InstallConfig) *field.Error {
 		return nil
 	}
 
+	// skip validation when user configured dns is enabled
```

Comment:
Ditto: Should this test be added [here](https://github.com/openshift/installer/blob/master/pkg/asset/installconfig/platformprovisioncheck.go#L93) instead?